cime_config/testdefs/testmods_dirs/cam/outfrq_dme_adjust_derecho/user_nl_cam

```diff
@@ -0,0 +1,7 @@
+ncdata='/glade/campaign/cesm/community/amwg/sima_baselines/cam_sima_test_snapshots/cam_ne3pg3_dme_adjust_snapshot_derecho_gnu_before_c20250416.nc'
+ncdata_check='/glade/campaign/cesm/community/amwg/sima_baselines/cam_sima_test_snapshots/cam_ne3pg3_dme_adjust_snapshot_derecho_gnu_after_c20250416.nc'
+debug_output=0
+pver=32
+hist_add_inst_fields;h2: Q,T,U,V,PS
```

Comment:
I would only include `Q` and `T` here, as those are the only fields that `dme_adjust` actually modifies.